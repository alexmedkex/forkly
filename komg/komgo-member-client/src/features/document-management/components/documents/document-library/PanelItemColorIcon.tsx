import styled from 'styled-components'

import { categoryToColor } from './categoryToColor'
interface Props {
  categoryId: string
}
export const PanelItemColorIcon = styled.div`
  height: 0.5em;
  width: 0.5em;
  background-color: ${(props: Props) => categoryToColor[props.categoryId]};
  border-radius: 2px;
  align-self: center;
`
