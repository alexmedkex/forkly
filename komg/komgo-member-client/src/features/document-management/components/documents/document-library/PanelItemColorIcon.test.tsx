import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { PanelItemColorIcon } from './PanelItemColorIcon'

describe('PanelItemColorIcon', () => {
  it('renders', () => {
    expect(renderer.create(<PanelItemColorIcon categoryId="company-details" />).toJSON()).toMatchSnapshot()
  })
})
