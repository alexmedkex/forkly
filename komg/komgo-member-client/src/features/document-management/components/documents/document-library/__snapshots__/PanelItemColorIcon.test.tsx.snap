// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelItemColorIcon renders 1`] = `
.c0 {
  height: 0.5em;
  width: 0.5em;
  background-color: rgba(63,81,181);
  border-radius: 2px;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

<div
  className="c0"
/>
`;
